scilpy/io/streamlines.py

```diff
@@ -144,3 +146,52 @@ def load_tractogram_with_reference(parser, args, filepath,
         parser.error('{} is an unsupported file format'.format(filepath))
 
     return sft
+
+
+def filter_tractogram_data(tractogram, streamline_ids):
```

Comment:
I think this function should go in utils.streamlines, not in io!

Reply:
Moved !